packages/ui/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,84 @@
+import { composeEventHandlers } from "@layer-core/primitive";
+import { createContext } from "@layer-lib/react-context";
+import { HTMLAttributes, useState } from "react";
+
+interface TabsContextType {
+  defaultValue: string;
+  value?: string;
+  onValueChange: (val: string) => void;
+}
+
+const [TabsProvider, useTabsContext] = createContext<TabsContextType>();
+
+interface TabsProps extends React.PropsWithChildren, TabsContextType {}
+
+const Tabs = ({ defaultValue, onValueChange, children }: TabsProps) => {
+  const [value, setValue] = useState(defaultValue);
+  return (
+    <TabsProvider
+      defaultValue={defaultValue}
+      value={value}
+      onValueChange={(value: string) => {
+        onValueChange(value);
+        setValue(value);
+      }}
+    >
+      {children}
+    </TabsProvider>
+  );
+};
+
+type TabsListProps = HTMLAttributes<HTMLDivElement>;
+
+const TabsList = ({ children, ...props }: TabsListProps) => {
+  return (
+    <div aria-label="tabs-navigation" {...props}>
+      {children}
+    </div>
+  );
+};
+
+interface TabsTriggerProps extends HTMLAttributes<HTMLButtonElement> {
+  value: string;
+}
+
+const TabsTrigger = ({ value, children, ...props }: TabsTriggerProps) => {
+  const context = useTabsContext();
+  const isSelected = context.value === value;
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === "Enter" || event.key === " ") {
+      event.preventDefault();
+      context.onValueChange?.(value);
+    }
+  };
+  return (
+    <button
+      role="tab"
+      aria-controls={value}
+      aria-selected={context.value === value}
+      data-state={isSelected ? "active" : "inactive"}
+      onClick={() => context.onValueChange(value)}
+      onKeyDown={composeEventHandlers(handleKeyDown, props.onKeyDown ?? (() => {}))}
```

Comment:
```suggestion
      onKeyDown={composeEventHandlers(props.onKeyDown, (e) => {
          if (e.key === 'Enter' || e.key === ' ') {
            context.onValueChange(value)
          }
      })}
```

핸들러가 컴포넌트의 중심 구현사항은 아닌 거 같아서 핸들러 선언없이 요렇게 적는 거 어떠신가요 ?!

참고로 첫 번째 인자로 `originalEventHandler` 넣어야 합니다! 🙏